api/src/analysis.rs

```diff
@@ -759,7 +759,7 @@ impl<'a> deno_graph::source::Loader for GcsLoader<'a> {
 }
 
 #[derive(Default)]
-struct ModuleAnalyzer {
+pub struct ModuleAnalyzer {
```

Comment:
Not needed?

Reply:
It's needed for the tests https://github.com/jsr-io/jsr/blob/js-imports/api/src/npm/tarball.rs#L382